src/aap_eda/services/project/git.py

```diff
@@ -99,6 +99,7 @@ def clone(
         *,
         credential: Optional[Credential] = None,
         depth: Optional[int] = None,
+        verify_ssl: Optional[bool] = True,
```

Comment:
Type hint would be incorrect, this param should be just a boolean. 
```suggestion
        verify_ssl: bool = True,
```